mne/dipole.py

```diff
@@ -211,7 +213,6 @@ def __len__(self):
         """Handle len function"""
         return self.pos.shape[0]
 
-
```

Comment:
Travis will probably complain about this removed line because PEP8 style guidelines call for two (consecutive) empty lines before new function def, and now there is only one